src/src/WebServer/HTML_wrappers.cpp

```diff
@@ -312,7 +312,7 @@ void html_add_script(bool defer) {
   if (defer) {
     addHtml(F(" defer"));
   }
-  addHtml(F(" type='text/JavaScript'>"));
+  addHtml(F(">"));
```

Comment:
After your change, the "text" has been reduced to a single char.
So it is better to use `addHtml('>');` which is of type `char` instead of a FlashString.
A char is much smaller and uses less resources.